src/plot.js

```diff
@@ -172,7 +172,7 @@ export function plot(options = {}) {
 
   // Allows e.g. the pointer transform to support viewof.
   context.dispatchValue = (value) => {
-    if (figure.value === value) return;
+    if (figure.value === value || selectionEquals(figure.value, value)) return;
```

Comment:
This feels overfit to the brush interaction; I would rather push this logic down into the brush interaction.